addons/project/wizard/project_share_wizard_template.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProjectShareWizardTemplate(models.TransientModel):
+    _name = 'project.share.wizard.template'
+    _description = 'Project Sharing Template'
+
+    partner_id = fields.Many2one('res.partner', string='recipient')
+    project_id = fields.Many2one('project.project', string='project')
+    is_portal = fields.Boolean(default=True)
```

Comment:
```suggestion
    partner_id = fields.Many2one('res.partner', export_string_translation=False)
    project_id = fields.Many2one('project.project', export_string_translation=False)
    is_portal = fields.Boolean(default=True, export_string_translation=False)
```
These fields are never displayed, right? In which case, their label is irrelevant, and we can also prevent them from being exported